builtin/aws/ecs/platform.go

```diff
@@ -1567,6 +1572,9 @@ type Config struct {
 	// Subnets to place the service into. Defaults to the subnets in the default VPC.
 	Subnets []string `hcl:"subnets,optional"`
 
+	// SecurityGroups to pass security groups to deployment.
```

Comment:
This should specify that it's the security group ids. Also, you need to add security_groups to the Documentation function below here.